src/compas/geometry/primitives/ellipse.py

```diff
@@ -63,6 +63,43 @@ def __init__(self, plane, major, minor):
         self.major = major
         self.minor = minor
 
+    @property
+    def DATASCHEMA(self):
+        import schema
+        return schema.Schema({
+            "plane": schema.And(
+                lambda x: len(x[0]) == 3 and all(isinstance(i, float) for i in x[0]),
```

Comment:
since this will be a veeeery common check, should we have a `compas.data.validations.is_vector3` helper function or similar?